pkg/logs/input/listener/tcp_test.go

```diff
@@ -18,8 +19,8 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/logs/pipeline/mock"
 )
 
-// FIXME: Use a randomly assigned port, but this means allowing '0' in the config.
-const tcpTestPort = 10512
+// use a randomly assigned port between 10000 and 65535.
+var tcpTestPort = 10000 + rand.Intn(55535)
```

Comment:
Why is the random port helping?

Reply:
It's not really related, but it's better to have random ports to avoid port collision if tests are running in parallel